src/scss/trumps/_utilities.scss

```diff
@@ -212,6 +212,9 @@
 .u-spacingTop {
     margin-top: spacing() !important;
 }
+    .u-spacingTop--minus {
```

Comment:
I meant a modifier of the `c-mediaElement--fullstack` class, so something like `c-mediaElement--fullstack--negativeTop` after the `.c-mediaElement--fullstack &` rule?

Reply:
Done 👍